tests/project_helpers.py

```diff
@@ -150,7 +171,7 @@ def venv_hash(self) -> str:
         The Python version currently used to run the tests is used to compute
         the hash and create the venv.
         """
-        dummy_script = self.venv_script_lines(Path("/dev/null"))
+        dummy_script = self.venv_script_lines(Path(os.devnull))
```

Comment:
With the above rewrite this should become:

```suggestion
        # Fake venv_path and python_exe to avoid unnecessary cache misses
        dummy_script = list(self.venv_script_lines(Path(os.devnull), "python"))
```

Reply:
Resolved here, to be addressed in #406.